docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
```

Comment:
There is a class for it: [`TableHDU`](https://github.com/astropy/astropy/blob/master/astropy/io/fits/hdu/table.py#L692) vs. [`BinTableHDU`](https://github.com/astropy/astropy/blob/master/astropy/io/fits/hdu/table.py#L785)